repctl/pkg/cmd/snapshot.go

```diff
@@ -145,6 +151,12 @@ func createSnapshot(configFolder, rgName, prefix, snNamespace, snClass string, v
 
 	rg.Annotations[prefix+"/snapshotNamespace"] = namespace
 	rg.Annotations[prefix+"/snapshotClass"] = snClass
+	rg.Annotations[prefix+"/snapshotStorageClass"] = storageClass
+	if createPVCtrue {
+		rg.Annotations[prefix+"/snapshotCreatePVC"] = "true"
+	} else {
+		rg.Annotations[prefix+"/snapshotCreatePVC"] = "false"
+	}
```

Comment:
```suggestion
	rg.Annotations[prefix+"/snapshotCreatePVC"] = createPVCtrue
```
Additionally, maybe rename `createPVCtrue` to `createPVC` to avoid confusion?